src/core_editor/editor.rs

```diff
@@ -196,6 +200,28 @@ impl Editor {
         Some(())
     }
 
+    /// ```
+    /// use reedline::{Editor, LineBuffer};
+    ///
+    /// let mut editor = Editor::default();
+    /// editor.line_buffer().set_buffer(String::from("a"));
+    /// editor.set_previous_lines(false);
+    /// editor.line_buffer().set_buffer(String::from("ab"));
+    /// editor.set_previous_lines(false);
+    /// editor.line_buffer().set_buffer(String::from("ab "));
+    /// editor.set_previous_lines(false);
+    /// editor.line_buffer().set_buffer(String::from("ab c"));
+    /// editor.set_previous_lines(true);
+    ///
+    /// assert_eq!(
+    ///     vec![
+    ///         LineBuffer::from(""),
+    ///         LineBuffer::from("ab "),
+    ///         LineBuffer::from("ab c")
+    ///     ],
+    ///     editor.get_edits()
+    /// );
+    /// ```
```

Comment:
@mzanrosso all I am proposing is that this doc test can be made into a unit test. That way we don't need to add the `editor.get_edits()` method to the public interface that `Editor` exposes. 
I will try to also add a PR as soon as I am able to.